.doc_gen/metadata/glue_metadata.yaml

```diff
@@ -637,6 +737,16 @@ glue_DeleteDatabase:
             - description:
               snippet_tags:
                 - javascript.v3.glue.actions.DeleteDatabase
+    C++:
```

Comment:
Is there no DeleteTable?

Reply:
That is not part of the scenario.  The table is deleted when the database is deleted.

Reply:
Ah, ok.